cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/compute_fleet_status/get_compute_fleet_status.erb

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+cat << EOF > get-compute-fleet-status.py
+import json
+import logging
+
+import argparse
+import boto3
+
+DB_KEY = "COMPUTE_FLEET"
+DB_DATA = "Data"
+
+COMPUTE_FLEET_STATUS_ATTRIBUTE = "status"
+COMPUTE_FLEET_LAST_UPDATED_TIME_ATTRIBUTE = "lastStatusUpdatedTime"
+
+log = logging.getLogger()
```

Comment:
Q where this is logging? do we want an additional log under `/var/log/parallelcluster/`?

Reply:
Change to output to stdout and error to stderror